### PR TITLE
fix: restricted column long value to 3 line and line clamped

### PR DESCRIPTION
**Pull request by SagarRajput-7**
### Summary

Added `lineclamping` to the table panel column values - lines - 3, did not restricted them to any max-width (column) as columns are resizable

#### Related Issues / PR's

https://github.com/SigNoz/engineering-pod/issues/1691

### CODE REVIEW

File: frontend/src/periscope/components/LineClampedText/LineClampedText.tsx [hunk 1]
```diff
@@ -42,11 +44,25 @@ function LineClampedText({
 		</div>
 	);
 
-	return isOverflowing ? <Tooltip title={text}>{content}</Tooltip> : content;
+	return isOverflowing ? (
+		<Tooltip
+			title={text}
+			placement="right"
+			autoAdjustOverflow
```
Comment (vikrantgupta25) [inline frontend/src/periscope/components/LineClampedText/LineClampedText.tsx:51]: make sure this is not affecting the already existing instances of this

File: frontend/src/container/GridTableComponent/index.tsx [hunk 2]
```diff
@@ -116,7 +134,7 @@ function GridTableComponent({
 							}
 						>
 							<Space>
-								{text}
+								{handleLongText(text)}
```
[the remark this answers is not in the document]
Comment (vikrantgupta25) [reply frontend/src/container/GridTableComponent/index.tsx:137]: are we getting any perf benefits here from the `useCallback` ?

**Review by vikrantgupta25 — DISMISSED** [review]

File: frontend/src/container/GridTableComponent/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ahmadshaheer) [reply frontend/src/container/GridTableComponent/index.tsx:137]: +1

**Review by rahulkeswani101 — DISMISSED** [review]

File: frontend/src/periscope/components/LineClampedText/LineClampedText.tsx (hunk 1 above)
Comment (ahmadshaheer) [reply frontend/src/periscope/components/LineClampedText/LineClampedText.tsx:51]: +1

**Review by ahmadshaheer — APPROVED** [review]

**Review by YounixM — DISMISSED** [review]

File: frontend/src/periscope/components/LineClampedText/LineClampedText.tsx (hunk 1 above)
Comment (SagarRajput-7) [reply frontend/src/periscope/components/LineClampedText/LineClampedText.tsx:51]: thanks for catching it, actually I forgot to remove these, after passing` ...tooltipProps`

**Review by rahulkeswani101 — DISMISSED** [review]

**Review by YounixM — DISMISSED** [review]

**Review by ahmadshaheer — APPROVED** [review]

**Review by vikrantgupta25 — DISMISSED** [review]
LGTM

File: frontend/src/container/NewDashboard/DashboardVariablesSelection/VariableItem.tsx [hunk 3]
```diff
@@ -254,6 +254,7 @@ function VariableItem({
 	});
 
 	const handleChange = (value: string | string[]): void => {
+		console.log('handleChange', value, variableData, optionsData);
```
Comment (vikrantgupta25) [inline frontend/src/container/NewDashboard/DashboardVariablesSelection/VariableItem.tsx:257]: remove console.log

Comment (SagarRajput-7) [reply frontend/src/container/NewDashboard/DashboardVariablesSelection/VariableItem.tsx]: done, Thanks!

**Review by rahulkeswani101 — APPROVED** [review]

**Review by YounixM — APPROVED** [review]

**Pull request merged**